internal/js/modules/k6/browser/common/page_options.go

```diff
@@ -67,6 +73,39 @@ func NewPageReloadOptions(defaultWaitUntil LifecycleEvent, defaultTimeout time.D
 	}
 }
 
+// NewPageGoBackForwardOptions returns a new PageGoBackForwardOptions.
+func NewPageGoBackForwardOptions(
+	defaultWaitUntil LifecycleEvent,
+	defaultTimeout time.Duration,
+) *PageGoBackForwardOptions {
+	return &PageGoBackForwardOptions{
+		WaitUntil: defaultWaitUntil,
+		Timeout:   defaultTimeout,
+	}
+}
+
+// Parse parses the page go back/forward options.
+func (o *PageGoBackForwardOptions) Parse(ctx context.Context, opts sobek.Value) error {
+	rt := k6ext.Runtime(ctx)
+	if !common.IsNullish(opts) {
```

Comment:
Please use early return here:

```go
if common.IsNullish(opts) {
    return nil
}
...
```

We also don't need `rt` to be a variable and we can pass it directly to `ToObject`.